app/views/organisers/index.html.slim

```diff
@@ -5,6 +5,13 @@
       p
       p
       p
+      = form_tag organisers_path, :method => :post do
```

Comment:
Ill validate once i write them :D I have completed first the main work. Now I will do this.